plugins/modules/aci_fabric_wide_settings.py

```diff
@@ -0,0 +1,318 @@
+    type: bool
+  leaf_opflex_client_auth:
+    description:
+    - Whether to enforce Opflex client certificate authentication on leaf switches for GOLF and Linux.
+    - The APIC defaults to C(true) when unset during creation.
+    type: bool
+  spine_ssl_opflex:
+    description:
+    - Whether to enable SSL Opflex transport for spine switches.
+    - The APIC defaults to C(true) when unset during creation.
+    type: bool
+  leaf_ssl_opflex:
+    description:
+    - Whether to enable SSL Opflex transport for leaf switches.
+    - The APIC defaults to C(true) when unset during creation.
+    type: bool
+  opflex_ssl_versions:
+    description:
+    - Which versions of TLS to enable for Opflex.
+    - When setting any of the TLS versions, you must explicitly set the state for all of them.
+    type: list
+    elements: str
+    choices: [ tls_v1.0, tls_v1.1, tls_v1.2 ]
+  reallocate_gipo:
+    description:
+    - Whether to reallocate some non-stretched BD gipos to make room for stretched BDs.
+    - Asking for gipo reallocation is a one time operation. Once enabled, it cannot be disabled.
+    - The APIC defaults to C(false) when unset during creation.
+    type: bool
+  restrict_infra_vlan_traffic:
+    description:
+    - Whether to restrict infra VLAN traffic to only specified network paths. These enabled network paths are defined by infra security entry policies.
+    - The APIC defaults to C(false) when unset during creation.
+    type: bool
+  state:
+    description:
+    - Use C(present) for updating configuration.
+    - Use C(query) for showing current configuration.
+    type: str
+    choices: [ present, query ]
+    default: present
+extends_documentation_fragment:
+- cisco.aci.aci
+- cisco.aci.annotation
+- cisco.aci.owner
+
+seealso:
+- name: APIC Management Information Model reference
+  description: More information about the internal APIC class B(infra:SetPol).
+  link: https://developer.cisco.com/docs/apic-mim-ref/
+author:
+- Tim Cragg (@timcragg)
+- Akini Ross (@akinross)
+"""
+
+EXAMPLES = r"""
+- name: Update Fabric Wide Settings
+  cisco.aci.aci_fabric_wide_settings:
+    host: apic
+    username: admin
+    password: SomeSecretPassword
+    disable_remote_ep_learning: true
+    enforce_epg_vlan_validation: true
+    state: present
+  delegate_to: localhost
+
+- name: Update Opflex SSL versions
+  cisco.aci.aci_fabric_wide_settings:
+    host: apic
+    username: admin
+    password: SomeSecretPassword
+    opflex_ssl_versions: [ tls_v1.2 ]
+    state: present
+  delegate_to: localhost
+
+- name: Query Fabric Wide Settings
+  cisco.aci.aci_fabric_wide_settings:
+    host: apic
+    username: admin
+    password: SomeSecretPassword
+    state: query
+  delegate_to: localhost
+  register: query_result
+"""
+
+RETURN = r"""
+current:
+  description: The existing configuration from the APIC after the module has finished
+  returned: success
+  type: list
+  sample:
+    [
+        {
+            "fvTenant": {
```

Comment:
Another one: fvTenant or infraSetPol?

Reply:
Okay I realised that the dev guide mentions to just copy and use the same example.